charts/rancher-istio/104.4.1+up1.22.1/Chart.yaml

```diff
@@ -11,14 +11,14 @@ annotations:
   catalog.cattle.io/requests-memory: 2314Mi
   catalog.cattle.io/type: cluster-tool
   catalog.cattle.io/ui-component: istio
-  catalog.cattle.io/upstream-version: 1.18.2
+  catalog.cattle.io/upstream-version: 1.22.1
 apiVersion: v1
-appVersion: 1.18.2
+appVersion: 1.22.1
 description: A basic Istio setup that installs with the istioctl. Refer to https://istio.io/latest/
   for details.
 icon: https://charts.rancher.io/assets/logos/istio.svg
 keywords:
 - networking
 - infrastructure
 name: rancher-istio
-version: 1.18.2
+version: 104.4.1+up1.22.1
```

Comment:
Are we changing the versioning?

Reply:
Not really sure why git is comparing these two files, this is a forward-port of [this](https://github.com/rancher/charts/blob/dev-v2.9/charts/rancher-istio/104.4.1%2Bup1.22.1/Chart.yaml), so we are not changing the versioning.